client-node.py

```diff
@@ -260,20 +354,24 @@ def initializeGUI():
     connectButton = Button(root, text="Connect", command=connectToIP)
     connectButton.pack(side=RIGHT)
 
-
-
-    #listB.pack()
-
     return root
 
+#start a server for the chat service and GUI
+factory = protocol.ServerFactory()
+factory.protocol = EchoServerProtocol
+try:
+    reactor.listenTCP(9000, factory)
+except: #won't break absolutly everything if you run two instances on one machine
+    print("Error starting Chat Server: port in use")
```

Comment:
Instead of doing a print here. Try using the logger. log.err() if you have an error.  That way, if later on, we decide to output to a logfile instead, all of the errors will be dumped there.